tests/specs/lemmas.k

```diff
@@ -147,7 +147,7 @@ module LEMMAS
     // for terms came from bytecode not via #hashedLocation
     rule keccak(WS) => keccakIntList(byteStack2IntList(WS))
```

Comment:
byteStack2IntList is used here but it is removed from edsl.md.

@daejunpark I think we should keep `keccak(WS)` as the canonical term in the configuration instead of converting it to `keccakIntList`. is there any specific reason that we want to do it?

Reply:
It is better to keep this for now, unless we cannot put `keccakIntList` back in the edsl.md.

Your suggestion is not infeasible, but requires changing existing proofs. Let's keep this PR minimal.

Reply:
I've changed it back to how it was before.